fitlins/workflows/base.py

```diff
@@ -59,17 +60,37 @@ def init_fitlins_wf(bids_dir, derivatives, out_dir, analysis_level, space,
         name='getter')
 
     if smoothing:
-        smoothing_params = smoothing.split(':', 1)
-        if smoothing_params[0] != 'iso':
-            raise ValueError(f"Unknown smoothing type {smoothing_params[0]}")
-        smoothing_fwhm = float(smoothing_params[1])
+        smoothing_params = smoothing.split(':', 2)
+        smoothing_fwhm = float(smoothing_params[0])
+        if smoothing_fwhm == 'iso':
+            warnings.warn(
+                "The order of arguments for smoothing will"
+                "change in the next release.", FutureWarning)
+            smoothing_level = 'l1'
+            smoothing_fwhm = smoothing_params[1]
+        else:
+            if len(smoothing_params) > 1:
+                smoothing_level = smoothing_params[1]
+            else:
+                smoothing_level = 'l1'
+
+            if len(smoothing_params) > 2:
+                if smoothing_params[2] != 'iso':
+                    raise ValueError(
+                        f"Unknown smoothing type {smoothing_params[1]}")
```

Comment:
Again have a bunch of little comments, so here's a replacement for L63-L80:

```Python
        smoothing_params = smoothing.split(':', 2)
        # Convert old style and warn; this should turn into an (informative) error around 0.5.0
        if smoothing_params[0] == 'iso':
            smoothing_params = (smoothing_params[1], 'l1', smoothing_params[0])
            warnings.warn(
                "The format for smoothing arguments has changed. Please use "
                f"{':'.join(smoothing_params)} instead of {smoothing}.", FutureWarning)
        # Add defaults to simplify later logic
        if len(smoothing_params) == 1:
            smoothing_params.extend(('l1', 'iso'))
        elif len(smoothing_params) == 2:
            smoothing_params.append('iso')

        smoothing_fwhm, smoothing_level, smoothing_type = smoothing_params
        smoothing_fwhm = float(smoothing_fwhm)
        if smoothing_type not in ('iso'):
            raise ValueError(f"Unknown smoothing type {smoothing_type}")